service/volume.go

```diff
@@ -22,7 +23,9 @@ func (s *Service) DeleteVolumes(c container.Container) error {
 			wg.Add(1)
 			go func(source string) {
 				defer wg.Done()
-				if err := c.DeleteVolume(source); err != nil {
+				// if service is never started before, data volume won't be created and Docker Engine
+				// will return with the not found error. therefore, we can safely ignore it.
+				if err := c.DeleteVolume(source); err != nil && !client.IsErrNotFound(err) {
```

Comment:
could be done directly in the `DeleteVolume` function that ignores the error to avoid the dependency with docker but it's fine like that too